packages/remix-deno/globals.ts

```diff
@@ -10,8 +10,10 @@ declare global {
 
   // This is here because we need it in our node setup,
   // but not in an actual deno project.
+  /* eslint-disable prefer-let/prefer-let */
```

Comment:
Seems like we should open an issue on the `prefer-let` code to explain this use case to them and ask them for the ability to opt-out in global TS declarations?